lib/appsignal/config.rb

```diff
@@ -329,6 +331,14 @@ def write_to_environment # rubocop:disable Metrics/AbcSize
     end
 
     def validate
+      skip_session_data = config_hash[:skip_session_data]
+      unless skip_session_data.nil?
+        merge(:send_session_data => !skip_session_data)
```

Comment:
If I'm reading this correctly, this means `skip_` has priority over `send_` -- as in, if you set both, the value determined by `skip_` will prevail. Not sure if this would be a problem in practice.